File: php/standard_deviation.php
<?php

//// algorithms for Standard Deviation Calculator in PHP 
$x=[]; 
$sum = 0; 
$n = (int)readline("enter the lenght of array");
$variance=0;
for($i=0;$i<$n;$i++)
{
    $x[$i] =(int)readline("enter the valuse for the index number".$i+1);
    $sum += $x[$i];
}
$avg = $sum/ $n;
for($i = 0 ; $i<$n; $i++)
{
    $variance += pow(($x[$i]-$avg),2);
} 
$variance /= $n;
echo $s =  (float)sqrt($variance);

File: c++/standard_deviation.cpp
#include <iostream>
#include <cmath>
using namespace std;

/// algorithms for Standard Deviation Calculator in cpp 

float standard_deviation(int n, int size[]) {
    float sum = 0, variance = 0, avg = 0, stdev = 0;

    for (int i = 0; i < n; i++) {
        sum += size[i];
    }
    avg = sum / n;

    for (int i = 0; i < n; i++) {
        variance += pow(size[i] - avg, 2);
    }
    variance /= n;

    stdev = sqrt(variance);

    return stdev;
}

int main() {
    int n; 
    cout << "Enter the size of the array: ";
    cin >> n;
    int size[n];
   
    for (int i = 0; i < n; i++) {
        cout << "Enter element " << i + 1 << ": ";
        cin >> size[i];
    } 

    float result = standard_deviation(n, size);

    cout << "The standard deviation is: " << result << "\n";

    return 0;
}